lesson-01-servlet-api/src/main/java/ru/geekbrains/persist/UserRepository.java

```diff
@@ -0,0 +1,38 @@
+package ru.geekbrains.persist;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class UserRepository {
+
+    private Map<Long,User> userMap = new ConcurrentHashMap<>();
+
+    private AtomicLong identity = new AtomicLong(0);
+
+    public List<User> findAll(){
+        return new ArrayList<>(userMap.values());
+    }
+
+    public User findById(long id){
+        return userMap.get(id);
+    }
+
+    public void Insert(User user){
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.